databricks/koala/tests/test_dataframe.py

```diff
@@ -130,6 +130,31 @@ def test_index_names(self):
         ddf = self.spark.from_pandas(df)
         self.assertEqual(ddf.index.name, 'x')
 
+    def test_rename(self):
+        df = pd.DataFrame({'a': [1, 2, 3, 4, 5, 6, 7],
+                           'b': [7, 6, 5, 4, 3, 2, 1]})
+        ddf = self.spark.from_pandas(df)
+
+        ddf2 = ddf.rename(columns={'a': 'x'})
+        df2 = df.rename(columns={'a': 'x'})
+        self.assert_eq(ddf2.columns, pd.Index(['x', 'b']))
+        self.assert_eq(ddf2, df2)
+
+        if pd.__version__ > '0.21':
```

Comment:
we can assume this is the case based on the package bounds.

Reply:
No need to check for it explicitly

Reply:
I had to add this because in Travis there is a check for Spark 2.3 and Pandas 0.19